ssl/record/ssl3_record.c

```diff
@@ -1385,14 +1386,13 @@ int tls1_mac(SSL *ssl, SSL3_RECORD *rec, unsigned char *md, int sending)
                 break;
         }
     }
-#ifdef SSL_DEBUG
-    {
+    if (OSSL_debug_is_set(OSSL_DEBUG_SSL)) {
         unsigned int z;
+
         for (z = 0; z < md_size; z++)
-            fprintf(stderr, "%02X ", md[z]);
-        fprintf(stderr, "\n");
+            OSSL_debug(OSSL_DEBUG_SSL, "%02X ", md[z]);
+        OSSL_debug(OSSL_DEBUG_SSL, "\n");
```

Comment:
This should somehow be identified as MD data.

Reply:
Yup, I added a small lable